.env.example

```diff
@@ -26,3 +26,8 @@ POKT_PORTAL_SECRET=
 OPENSEA_API_KEY=
 
 SENTRY_DSN=
+
+AWS_KEY=
+AWS_SECRET= 
+AWS_REGION=
+AWS_ETH_XYZ_HOSTED_ZONE_ID=
```

Comment:
@makhnovskiy can you rename this to `AWS_HOSTED_ZONE_ID` instead? The reason is, the project is open source so anyone can take it a put it into their own domain so I would prefer to keep this as "brandless/standard" as  possible.